audiobook_generator/tts_providers/openai_tts_provider.py

```diff
@@ -0,0 +1,91 @@
+import io
+import logging
+import math
+
+from openai import OpenAI
+
+from audiobook_generator.core.audio_tags import AudioTags
+from audiobook_generator.config.general_config import GeneralConfig
+from audiobook_generator.core.utils import split_text, set_audio_tags
+from audiobook_generator.tts_providers.base_tts_provider import BaseTTSProvider
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_supported_models():
+    return ["tts-1", "tts-1-hd"]
+
+
+def get_supported_voices():
+    return ["alloy", "echo", "fable", "onyx", "nova", "shimmer"]
+
+
+def get_supported_formats():
+    return ["mp3", "acc", "flac"]
```

Comment:
I think the `opus` option could also be supported. I remember I have tested it. https://platform.openai.com/docs/guides/text-to-speech/supported-output-formats

Reply:
yes, missed, fixing